west.yml

```diff
@@ -42,10 +42,8 @@ manifest:
       revision: e3a03f5ec7d8d33be705c5ce8a632d998ce9b4d1
 
     - name: zephyr
-      repo-path: zephyr
-      revision: 1851950977d5c89d8727e1058a5732f71ce4cd2a
-      remote: zephyrproject
-
+      url: https://github.com/aborisovich/zephyr
```

Comment:
honestly, you don't want to use your zephyr fork on main 
Adding "req change" - to prevent somebody from accidently merging this

Reply:
This is obviously for testing. I hope no maintainer would merge this but if you want to be extra safe making this a "draft" is better than [DNM]. Drafts have no "merge" button at all.